com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ScreenshotUtilities.cs

```diff
@@ -15,41 +16,45 @@ namespace Microsoft.MixedReality.GraphicsTools.Editor
     /// </summary>
     public class ScreenshotUtilities
     {
+        [Shortcut("Graphics Tools/Take Screenshot 1x", KeyCode.Alpha1, ShortcutModifiers.Alt)]
         [MenuItem("Window/Graphics Tools/Take Screenshot/Native Resolution")]
         private static void CaptureScreenshot1x()
         {
             CaptureScreenshot(GetScreenshotPath(), 1);
             EditorUtility.RevealInFinder(GetScreenshotDirectory());
         }
 
+        [Shortcut("Graphics Tools/Take Screenshot 1x Alpha", KeyCode.Alpha1, ShortcutModifiers.Shift)]
         [MenuItem("Window/Graphics Tools/Take Screenshot/Native Resolution (Transparent Background)")]
         private static void CaptureScreenshot1xAlphaComposite()
         {
             CaptureScreenshot(GetScreenshotPath(), 1, true);
             EditorUtility.RevealInFinder(GetScreenshotDirectory());
         }
-
+        [Shortcut("Graphics Tools/Take Screenshot 2x", KeyCode.Alpha2, ShortcutModifiers.Alt)]
         [MenuItem("Window/Graphics Tools/Take Screenshot/2x Resolution")]
         private static void CaptureScreenshot2x()
         {
             CaptureScreenshot(GetScreenshotPath(), 2);
             EditorUtility.RevealInFinder(GetScreenshotDirectory());
         }
 
+        [Shortcut("Graphics Tools/Take Screenshot 2x Alpha", KeyCode.Alpha2, ShortcutModifiers.Shift)]
         [MenuItem("Window/Graphics Tools/Take Screenshot/2x Resolution (Transparent Background)")]
         private static void CaptureScreenshot2xAlphaComposite()
         {
             CaptureScreenshot(GetScreenshotPath(), 2, true);
             EditorUtility.RevealInFinder(GetScreenshotDirectory());
         }
-
+        [Shortcut("Graphics Tools/Take Screenshot 4x", KeyCode.Alpha4, ShortcutModifiers.Alt)]

```

Comment:
Nitpick: newline between the previous method and attribute:

```suggestion

        [Shortcut("Graphics Tools/Take Screenshot 4x", KeyCode.Alpha4, ShortcutModifiers.Alt)]
```